tools/VscPreviewExtension/CHANGELOG.md

```diff
@@ -0,0 +1,55 @@
+## Known Issues
+* Preview match: File inclusion
+* Match when multiple open editor
+
+## Current Version **0.0.21**
```

Comment:
Move to top? How about:

# Current Version...
## New Features
...
## Known Issues
...

## Version History
...